sinch/core/ports/http_transport.py

```diff
@@ -29,6 +46,23 @@ def authenticate(self, endpoint, request_data):
                 "Authorization": f"Bearer {token}",
                 "Content-Type": "application/json"
             })
+        elif endpoint.HTTP_AUTHENTICATION == HTTPAuthentication.SIGNED.value:
```

Comment:
_same comment related to validation at domain level_

Reply:
same comment related to validation at domain level :)